tools/enso4igv/src/main/java/org/enso/tools/enso4igv/EnsoRootProject.java

```diff
@@ -88,6 +89,9 @@ public Set<? extends Project> getSubprojects() {
     }
 
     private static void searchForProjects(FileObject fo, Collection<Project> found, int depth) {
+      if (fo.getName().startsWith("bazel")) {
```

Comment:
There was a lot of "duplicated" projects found (like `aws`, `base`, `benchmarks`):

![duplications](https://github.com/user-attachments/assets/93014230-b3f7-49a8-834b-a3efe664e9de)

Turns out those projects were created by Bazel as copies in `bazel` subdirectories. This check skips the Bazel subdirectories and adds a test to verify the found projects roughly match the defined ones: 4086402ee8d3db8c3c82c61bf2508c6b26dd5594

Reply:
Yes, it's unfortunate.